packages/mock-repo/src/composite/composite.tsx

```diff
@@ -1,11 +1,13 @@
 import * as React from 'react';
 import {ChildComp} from './child';
+import style from './child.st.css';
 
 export interface IProps {
   text?: React.ReactNode;
+  className?: string;
 }
 
 export const Composite: React.SFC<IProps> = (props) =>
-  <div><ChildComp text={props.text} /></div>;
+  <div  {...style('root', {}, props)} className={props.className}><ChildComp text={props.text} /></div>;
```

Comment:
You shouldn't need to add `className` explicitly, `...style()` should do that for you.